src/test/java/br/ufc/quixada/controller/HistoricControllerTest.java

```diff
@@ -0,0 +1,64 @@
+package br.ufc.quixada.controller;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.sql.SQLException;
+import java.util.concurrent.TimeoutException;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.testfx.api.FxRobot;
+import org.testfx.api.FxToolkit;
+import org.testfx.framework.junit5.ApplicationTest;
+
+import br.ufc.quixada.dao.MatchHistoryDAO;
+import br.ufc.quixada.util.SceneManager;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Node;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+public class HistoricControllerTest extends ApplicationTest {
+
+    private MatchHistoryDAO mockMatchHistoryDAO;
+    private HistoricController controller;
+
+    @Override
+    public void start(Stage stage) throws Exception {
+        FXMLLoader loader = new FXMLLoader(getClass().getResource("/br/ufc/quixada/fxml/historic.fxml"));
+        Parent root = loader.load();
+        controller = loader.getController();
+
+        // Substitua o DAO real por um mock
+        mockMatchHistoryDAO = Mockito.mock(MatchHistoryDAO.class);
```

Comment:
Para substituir o DAO por um mock é preciso adicionar um método set no controller, o mock também precisa retornar uma lista de MatchHistory quando o método `list` for chamado, mockito possui método `when` e `thenReturn` para isso:

```java
when(mockMatchHistoryDAO.list()).thenReturn(/*Lista de mock*/);
```